blocks/image-placeholder/index.js

```diff
@@ -38,13 +38,15 @@ export default function ImagePlaceHolder( { className, icon, label, onSelectImag
 			>
 				{ __( 'Upload' ) }
 			</FormFileUpload>
-			<MediaUploadButton
-				buttonProps={ { isLarge: true } }
+			<MediaUpload
 				onSelect={ onSelectImage }
 				type="image"
-			>
-				{ __( 'Add from Media Library' ) }
-			</MediaUploadButton>
+				render={ ( { open } ) => (
```

Comment:
"Children as function" would also work but I personally prefer "render props". I prefer if children would have been always used as "arrays" to avoid confusion in the shape of Children in general.

Reply:
I heard in Egghead.io podcast that `render props` is easier to follow for the majority of developers. I like it more than this awkward `this.props.children()`.